### ISSUE TITLE
Add example of saving

### ISSUE THREAD

**Issue by hyanwong** [jthlab/phlash#8]
For running on a cluster, it could be helpful to show how to save and reload results from the `fit` function (i.e. a list of `DemographicModel` objects). E.g. can these simply be `pickled`, or would that take up too much space?

More generally, what's the recommended way of saving output from long runs?

**Comment by terhorst** [comment]
Yes, they can simply be pickled, and that is the intended usage. DemographicModel is just a NamedTuple (so a tuple) and requires minimal space (about  `3 * time points * sizeof(float)` to store) so there should be no issue with saving a large number of them.

Will add to docs!
